TypeScript/confirmation.ts

```diff
@@ -0,0 +1,138 @@
+//@ts-nocheck
+/**
+ * @author @HighArcs [<@504698587221852172>]
+ * @version 1.0.0
+ * @description asks the user for a confirmation message before executing the command
```

Comment:
description should not be written like this, please refer other plugin files for correct examples